src/examples/poi-crosschecker/utils.ts

```diff
@@ -71,47 +91,32 @@ export const printNPOIs = (nPOIs: Map<string, Map<string, Attestation[]>>) => {
       console.log(`🔍  Attestations for block ${block}:`.cyan);
       attestations.forEach((a) => {
         console.log(
-          `nPOI: ${a.nPOI}\nSender: ${a.indexerAddress}\nStake:${a.stakeWeight}\n`
+          `nPOI: ${a.nPOI}\nSender: ${a.operator}\nStake:${a.stakeWeight}\n`
             .cyan
         );
       });
     });
   });
 };
 
-export const sortAttestations = (attestations: Attestation[]) => {
+//TODO: modify attestation types
+export const sortAttestations = (records: NPOIRecord[]) => {
   const groups = [];
-  attestations.forEach((attestation: Attestation) => {
+  records.forEach((record: NPOIRecord) => {
     // if match with group's nPOI, update that group
-    const matchedGroup = groups.find((g) => g.nPOI === attestation.nPOI);
+    const matchedGroup = groups.find((g) => g.nPOI === record.nPOI);
     if (matchedGroup) {
-      matchedGroup.stakeWeight += attestation.stakeWeight;
-      matchedGroup.indexers.push(attestation.indexerAddress);
+      matchedGroup.stakeWeight += record.stakeWeight;
+      matchedGroup.indexers.push(record.operator);
     } else {
       groups.push({
-        nPOI: attestation.nPOI,
-        stakeWeight: attestation.stakeWeight,
-        indexers: [attestation.indexerAddress],
+        nPOI: record.nPOI,
+        stakeWeight: record.stakeWeight,
+        indexers: [record.operator],
       });
     }
   });
 
-  const sorted = groups.sort((a, b) => Number(b.stakeWeight - a.stakeWeight));
+  const sorted = groups.sort((a, b) => Number(a.stakeWeight - b.stakeWeight));
```

Comment:
I changed sorting here because I thought this is a bug - `x.sort((a,b) => a-b)` is ascending, `x.sort((a,b) => b-a)` is descending. Does this behave differently for you?

Reply:
ah huge thanks for noticing this! I must have been overwritten after I merged main